test/utilities.ts

```diff
@@ -0,0 +1,53 @@
+import Web3 from "web3";
+import {HttpProvider} from "web3-core";
+import {EpochTokenLockerInstance} from "../types/truffle-typings";
+
+const jsonrpc = "2.0";
+const id = 0;
+const send = function (method: string, params: any, web3Provider: Web3) {
+  return new Promise(function (resolve, reject) {
+    (web3Provider.currentProvider as HttpProvider).send(
+      {id, jsonrpc, method, params},
+      (error, result) => {
+        if (error) {
+          reject(error);
+        } else {
+          resolve(result);
+        }
+      }
+    );
+  });
+};
+
+// Wait for n blocks to pass
+/**
+ * Wait for n (evm) seconds to pass
+ * @param seconds: int
+ * @param web3Provider: potentially different in contract tests and system end-to-end testing.
+ */
+export const waitForNSeconds = async function (
```

Comment:
I'm curious why you prefer this syntax over `export async function waitForNSeconds(`.

Reply:
This was probably written by me who doesn't know anything about JS. That swapping all this stuff around randomly and still work frightens me.

Reply:
I don't prefer one over the other but just prepending export was the minimal distance to make it compile.

Reply:
I say then, if we decide to change it, it should be done in a separate PR.